### ISSUE TITLE
1.12 bug: keeps eating same food + tooltip fix

### ISSUE THREAD

**Issue by Flemmli97** [squeek502/SpiceOfLife#144]
version 1.3.11 in mc version 1.12:
sometimes when i try to eat food in the lunch box it keeps eating the same food so diminishing returns goes to 0%. client side sais full hunger but server not so when the hunger updates its back to what it was before eating all that food. items are gone though.

don't know of a way to reproduce it, but seems to happen more often with food which has high hunger and saturation value.

also since i don't know to create a pull request: you should add this to "GuiFoodContainer"

	@Override
	public void drawScreen(int mouseX, int mouseY, float partialTicks) {
		super.drawScreen(mouseX, mouseY, partialTicks);
        this.renderHoveredToolTip(mouseX, mouseY);
	}

it makes it render the tooltip which it currently doesn't.

**Comment by Xiaminou** [comment]
Yeah it would be nice to have the default inventory behavior when opening spice of life's food containers. Right now if I want to check out the tooltips of the food that are in them I have to pull them out and open the re-open my inventory, it's annoying.

**Comment by squeek502** [comment]
Tooltip bug should be fixed in 1.3.12. Unsure about the diminishing returns stuff.
